dropwizard-jdbi/src/main/java/io/dropwizard/jdbi/DBIFactory.java

```diff
@@ -53,7 +57,14 @@ public DBI build(Environment environment,
         final String validationQuery = configuration.getValidationQuery();
```

Comment:
@prb [Come on over](https://github.com/brettwooldridge/HikariCP), the water's nice.  We're fully Dropwizard enabled. :+1: